documentation/manual/src/main/asciidoc/en-US/modules/redis.asciidoc

```diff
@@ -717,10 +845,9 @@ public class GrandChild {
 
 [source, JSON]
 ----
-Key: GrandMother:86ada718-f2a2-4299-b6ac-3d90b1ef2331
-{
-    "grandChildren" : [ "Luke", "Leia" ]
-}
+Key: Associations:GrandMother_grandChildren:86ada718-f2a2-4299-b6ac-3d90b1ef2331:grandChildren
```

Comment:
Is it indeed the association table name ("GrandMother_grandChildren") or actually the entity table name ("GrandMother")? IIRC, we meant to go for the latter?